Revit_Core_Engine/Query/DuctSectionProperty.cs

```diff
@@ -40,21 +36,19 @@ public static partial class Query
         /****               Public Methods              ****/
         /***************************************************/
 
-        [Description("Duct section property converted from a Revit duct.")]
-        [Input("revitDuct", "Revit duct to be conerted into a section property.")]
+        [Description("Convert a Revit duct into a BHoM duct section property.")]
+        [Input("revitDuct", "Revit duct to be converted into a BHoM section property.")]
         [Input("settings", "Revit settings.")]
         [Input("refObjects", "Referenced objects.")]
-        [Output("DuctSectionProperty", "BHoM duct section property.")]
+        [Output("sectionProperty", "BHoM duct section property converted from a Revit duct.")]
```

Comment:
It is query/extraction, not conversion imo.

Reply:
Revised